src/forms/Checkbox.jsx

```diff
@@ -102,6 +102,7 @@ export class Checkbox extends React.PureComponent {
 					onChange={this.onChange}
 					readOnly={!this.onChange || disabled}
 					name={name}
+					aria-label={ariaLabel || 'checkbox'}
```

Comment:
Do we need additional prop for it? It will require to add it to all existing checkboxes.
I think we can just make it `aria-label={label || 'checkbox'}`

Reply:
🤔  make sense hah

Reply:
I will do it like this {`${label} - checkbox ` || 'checkbox'}